openpower/patches/p8dtu-patches/hostboot-p8/0003-VDDR-patch-for-p8dtu.patch

```diff
@@ -0,0 +1,49 @@
+From 783332cc9afcf3c8581760112162b650a9e01c7e Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Tue, 4 Oct 2016 16:07:44 -0700
+Subject: [PATCH 03/14] VDDR patch for p8dtu.
```

Comment:
What's the purpose of this patch? What problem is it addressing?

Reply:
The developer created this patch left the company. I have no idea of the history of this patch.

Reply:
I don't know how much longer we need to support B&S.  This one is going to be hard to find out why it is here and we should not try to remove it out at this late of the project.  I think we should go ahead let this one merge unless you see a potential problem @stewart-ibm.

Reply:
VDDR is the memory voltage.  This is how you turn the memory on.  We have a reference implementation in Hostboot but any variation from that has to be in a patch of some kind.  I think I made a suggestion on how this could be folded back into upstream with a CONFIG flag or something similar, but I'm not fundamentally opposed to this approach since every system seems to have its own unique power controls in the P8 timeframe.

Reply:
Okay, the patch probably needs a bit more description then, so people who aren't named @dcrowell77 can work out what it's for :)

Reply:
Description:
Adds support for memory voltage control specific to the p8dtu platform.